lib/pangea/widgets/chat/overlay_message.dart

```diff
@@ -119,83 +90,85 @@ class OverlayMessage extends StatelessWidget {
     );
 
     return Flexible(
-      child: Material(
-        color: noBubble ? Colors.transparent : color,
-        clipBehavior: Clip.antiAlias,
-        shape: RoundedRectangleBorder(
-          borderRadius: borderRadius,
-        ),
-        child: Container(
-          decoration: BoxDecoration(
-            borderRadius: BorderRadius.circular(
-              AppConfig.borderRadius,
-            ),
-          ),
-          padding: noBubble || noPadding
-              ? EdgeInsets.zero
-              : const EdgeInsets.symmetric(
-                  horizontal: 16,
-                  vertical: 8,
-                ),
-          constraints: BoxConstraints(
-            maxWidth: width ?? FluffyThemes.columnWidth * 1.25,
+      child: SingleChildScrollView(
```

Comment:
I like this! Can you add a little comment explaining why it was added? (to prevent long messages from going off screen)